### PR TITLE
Org auto populate

### PR DESCRIPTION
**Pull request by Vayras**
issue : https://github.com/stakwork/sphinx-tribes/issues/1320
loom video :https://www.loom.com/share/cccde42b4cbc41a0802d5f2d3eae022a

fix:
Getting the uuid of the org from url checking if the user has that organisation and appending it
Tested on Chrome
Thanks to @elraphty for helping me on this one

### CODE REVIEW

File: src/people/main/FocusView.tsx [hunk 1]
```diff
@@ -263,7 +277,22 @@ function FocusedView(props: FocusViewProps) {
...
+  const searchKey = uuid;
+  let variableToAppend = '';
+
+  const foundObject = userOrganizations.find((obj: any) => obj.value === searchKey);
+
+  if (foundObject) {
+    variableToAppend = foundObject.value;
+    console.log('Found and appended:', variableToAppend);
+  } else {
+    console.log('Key not found');
```
Comment (kevkevinpal) [inline src/people/main/FocusView.tsx:290]: dont think we need these logs and probably should not expose them to the user

File: src/people/main/FocusView.tsx [hunk 2]
```diff
@@ -263,7 +277,22 @@ function FocusedView(props: FocusViewProps) {
       props?.ReCallBounties();
   }
 
-  let initialValues: any = {};
+  //this workflow now gets the org user is in and appends it
+  const searchKey = uuid;
+  let variableToAppend = '';
+
+  const foundObject = userOrganizations.find((obj: any) => obj.value === searchKey);
```
Comment (kevkevinpal) [inline src/people/main/FocusView.tsx:284]: can we rename `foundObject` to something more unique to this scenario seems like pretty generic naming

**Comment by Vayras** [comment]
@kevkevinpal Sure, making these changes right away

File: src/people/main/FocusView.tsx [hunk 3]
```diff
@@ -60,6 +61,19 @@ function FocusedView(props: FocusViewProps) {
...
 
+  const getOrg = async () => {
+    if (uiStore.meInfo?.id) {
+      const res = await main.getUserDropdownOrganizations(uiStore.meInfo?.id);
+    }
+  };
+  getOrg();
+
+  function getUUIDFromURL(url: string) {
+    const parts = url.split('/');
```
Comment (kevkevinpal) [inline src/people/main/FocusView.tsx:72]: lets check if the url matches this pattern first and then return the last part
`{url}/org/bounties/{uuid}`

**Comment by Vayras** [comment]
@kevkevinpal updated the PR, added a regex check for url, removed console statements and renamed variables

**Comment by ecurrencyhodler** [comment]
Nice work. Tested and paid.

**Pull request merged**